src/main/java/school/lemon/changerequest/java/introduction/pr1/Bob.java

```diff
@@ -6,8 +6,10 @@
 public class Bob {
 
     public static String hey(String phrase) {
-        if (phrase.trim().isEmpty())
-            return "Fine";
+
+        if (phrase.replaceAll(" ", "").equals("")) {
```

Comment:
Your previous implementation `phrase.trim().isEmpty()` was better due to code readability